server/src/main/java/org/elasticsearch/indices/ExecutorNames.java

```diff
@@ -64,4 +64,10 @@ public record ExecutorNames(String threadPoolForGet, String threadPoolForSearch,
             throw new IllegalArgumentException(threadPoolForWrite + " is not a valid thread pool");
         }
     }
+
+    public boolean contains(String threadPoolName) {
```

Comment:
here is this new method used? I think that I don't follow what this method is expected to do and be used.

Reply:
Ooops - that's a leftover from the previous implementation. Removed it in 7c49cdae50ff00c084071e9d611bbb16fce87aa1